### PR TITLE
Encrypting Images (replacement of #1338)

### PR DESCRIPTION
**Pull request by guanbinrui**
Do some refactors on #1338.

+ [x] add shuffle image service
+ [x] remove external pkgs 
+ [x] resolve reviews from #1338
+ [ ] should collapse the attachments if successfully de-shuffled

### CODE REVIEW

**Comment by neruthes** [comment]
Will #1360 be included in this PR?

File: src/components/InjectedComponents/PostDialog.tsx [hunk 1]
```diff
@@ -88,21 +89,41 @@ export interface PostDialogUIProps
...
+        const reader = new FileReader()
+        reader.onload = () => {
+            if (typeof reader.result === 'string') {
+                resolve(null)
+            } else {
+                resolve(reader.result)
+            }
+        }
+        reader.onerror = reject
+        reader.readAsArrayBuffer(file)
```
Comment (septs) [inline src/components/InjectedComponents/PostDialog.tsx:123]:
```ts
await file.arrayBuffer()
```

File: packages/maskbook/src/_background_loader.1.ts [hunk 2]
```diff
@@ -35,5 +41,6 @@ Object.assign(globalThis, {
         decryptFromText,
         decryptFromImageUrl,
         sendTransaction,
+        decryptFromShuffledImage,
```
Comment (Jack-Works) [inline packages/maskbook/src/_background_loader.1.ts:44]: I'm afraid this need a rebase now

**Comment by Tedko** [comment]
@nazariyv in case u need to find wheres the pr.
need a rebase though

**Comment by Jack-Works** [comment]
I have rebased this PR to master but things have changed a lot, and there might be errors in it.

**Comment by Jack-Works** [comment]
this PR is quite lost in time and hard to rebase. I suggest to open a new one and migrate the changes onto the latest master

**Pull request closed without merging**